src/lib/SnackBar/index.ts

```diff
@@ -0,0 +1,98 @@
+import './styles.css';
+
+const DEFAULT_TIMEOUT = 2750;
+
+export interface SnackBarOptions {
+  message: string;
+  timeout?: number;
+  actionText?: string;
+  actionHandler?: () => boolean | null;
+}
+
+class SnackBar {
+  on?: () => void;
+  onhide?: () => void;
+  private options: SnackBarOptions;
+  private element: Element = document.createElement('div');
+  private text: Element = document.createElement('div');
+  private button: Element = document.createElement('button');
+  private parent: Element;
+  private showing = false;
+  private closeTimer?: number;
+
+  constructor (options: SnackBarOptions, parent: Element) {
+    this.options = options;
+    this.parent = parent;
+
+    this.element.className = 'snackbar';
+    this.element.setAttribute('aria-live', 'assertive');
+    this.element.setAttribute('aria-atomic', 'true');
+    this.element.setAttribute('aria-hidden', 'true');
```

Comment:
I don't think we need this, since the presence in the dom is controlling visibility.

Reply:
It's used to control the animation.